.buildkite/hooks/environment

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export PATH="/.pulumi/bin:$PATH"
+export PULUMI_ACCESS_TOKEN="$(buildkite-agent secret get PULUMI_ACCESS_TOKEN)"
+
+eval "$(pulumi env open cnunciato/default/personal --format shell)"
```

Comment:
I know this won't work, but I'm just leaving it in for reference. (This command fetches and applies the environment variables called out in the README.)